.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           command: npm run test
       - run:
           name: build
-          command: npm run build
+          command: CI=false npm run build
```

Comment:
please add a comment to explain why we do that, and also a link to the issue you found